index.js

```diff
@@ -64,19 +64,51 @@ menuLinkContactElement.addEventListener('click', () => {
 });
 
 // ----------------------- contact validation ---------------------------------
-const formElement = document.getElementById('contact-form');
+// const formElement = document.getElementById('contact-form');
 // const formNameElement = document.getElementById('name');
-const formEmailElement = document.getElementById('email');
+// const formEmailElement = document.getElementById('email');
 
-formElement.addEventListener('submit', (event) => {
-  const email = formEmailElement.value;
-  const errmessages = [];
-  if (email.toLowerCase() !== email) {
+const formElements = {
+  formElement: document.getElementById('contact-form'),
+  formNameElement: document.getElementById('name'),
+  formEmailElement: document.getElementById('email'),
+  commentElement: document.getElementById('comment-box'),
+  emailErrElement: document.getElementById('email-err'),
+};
+
+const formData = {
+  errmessages: [],
+  error: formElements.emailErrElement.value,
+};
+
+formElements.formElement.addEventListener('submit', (event) => {
+  formData.name = formElements.formNameElement.value;
+  formData.email = formElements.formEmailElement.value;
+  formData.commentElement = formElements.commentElement.value;
+
+  // const email = formData.formEmailElement.value;
+  // formData.errmessages = [];
```

Comment:
[OPTIONAL]Please it is important to follow microverse clean code policy. Please kindly remove unused code for it makes your code base look untidy and it may also lead to you experiencing errors.